apps/src/p5lab/P5LabView.jsx

```diff
@@ -102,6 +102,11 @@ class P5LabView extends React.Component {
     // we don't want students to be able to draw their own backgrounds in spritelab so if we're showing
     // backgrounds alone, we must be in spritelab and we should get rid of the draw your own option
     const canDraw = !this.props.isBackground;
+    // TODO(JillianK): Remove this param and all uses of it once new sprites have been added to the Spritelab animaion library.
+    // Jira context: https://codedotorg.atlassian.net/browse/STAR-1319
+    const categoryImagePathPrefix = this.props.spriteLab
+      ? '/blockly/media/gamelab/animation-previews-spritelab/category_'
```

Comment:
since the category previews are separate now, should the spritelab category images live within `apps/static/spritelab` now?